feedback-module/src/assets/styling_classnames.js

```diff
@@ -0,0 +1,21 @@
+export const BUTTON_STYLE =
+  "text-left border-transparent hover:bg-primary-light hover:text-primary hover:border-primary padding-10px mobile-lg:padding-x-205 mobile-lg:padding-y-2 mobile-lg:font-sans-lg border-05 hover:border-bottom-05 shadow-btn margin-y-1 mobile-lg:margin-y-2";
+export const BUTTON_RIGHT_STYLE = "float-right margin-x-0 clear-both";
+
+export const CHECKBOX_STYLE =
+  "text-primary bg-transparent mobile-lg:font-sans-xl font-sans-md  margin-y-2";
+
+export const HEADER_STYLE =
+  "radius-top-lg bg-primary height-4 mobile-lg:height-6 display-flex padding-x-10px padding-9px mobile-lg:padding-x-2 mobile-lg:padding-y-105 font-sans-check";
+
+export const MODULE_CONTAINER_STYLE =
+  "bg-primary-light radius-top-lg padding-x-0";
+export const SCREEN_CONTAINER_STYLE = "padding-x-6 padding-y-5 overflow-hidden";
+
+export const LABEL_STYLE =
+  "text-primary mobile-lg:font-sans-2xl font-sans-lg maxw-none margin-y-3";
+export const TEXTAREA_STYLE =
+  "border-05 border-primary bg-primary-light focus:border-width-6px focus:bg-primary-lighter";
+export const TEXTINPUT_STYLE = TEXTAREA_STYLE;
+
+export const P_STYLE = "mobile-lg:font-sans-xl font-sans-md text-normal";
```

Comment:
We could probably just set a global p style (similarly to how we set our h1 styling)